src/test/core.ts

```diff
@@ -11,6 +11,8 @@ import vnode, { VNode } from '../vnode'
 import htmlDomApi from '../htmldomapi'
 import { CreateHook, InsertHook, PrePatchHook, RemoveHook, InitHook, DestroyHook, UpdateHook } from '../hooks'
 
+const hasSvgClassList = 'classList' in SVGElement.prototype
```

Comment:
IE11 does not have a `classList` on `SVGElement`. So some tests would just be skipped in it. I'm OK with that.

Reply:
you can also load a classlist polyfill for this case

Reply:
Thanks, @jvanbruegge. I actually did that and then changed to this. Have a feeling that this is better. Not sure why.